src/audio/pipeline.c

```diff
@@ -205,17 +205,23 @@ static void pipeline_trigger_sched_comp(struct pipeline *p,
 
 	switch (cmd) {
 	case COMP_TRIGGER_PAUSE:
-	case COMP_TRIGGER_STOP:
-		trace_pipe_with_ids(p, "pipeline_trigger_sched_comp(): "
-				    "PAUSE/STOP");
 		pipeline_schedule_cancel(p);
 		p->status = COMP_STATE_PAUSED;
 		break;
+#if NO_XRUN_RECOVERY
```

Comment:
Please explain with more detail the real error this is fixing in the commit message.